.github/workflows/ci.yaml

```diff
@@ -0,0 +1,112 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  UNITY_VERSION: 2020.2.6
+  PROJECT_PATH: samples/unity-of-bugs
+
+jobs:
+
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+        with:
+          submodules: recursive
+
+      # Unity is about 3gb in size and takes 20 minutes to download and install.
+      # Caching still takes some time, but brings it down to about 5 minutes.
+      # Note: if you change the installation step, remember to update the trailing
+      # hash at the end of the cache key.
+      - name: Restore cached Unity installation
+        id: cache-unity
+        uses: actions/cache@v2
+        with:
+          path: C:/Program Files/Unity
+          key: ${{ env.UNITY_VERSION }}-bf30a
+
+      - name: Install Unity
+        run: |
+          choco install unity --version ${{ env.UNITY_VERSION }} --yes --no-progress
```

Comment:
I hope there's the equivalent `brew install` for the time we get iOS builds going.

Reply:
Can we not build iOS from Windows?